index.html

```diff
@@ -214,6 +214,23 @@ <h3 class="section-title">Getting Data Into Your Widgets</h3>
 
         <script src="https://gist.github.com/davefp/6112780.js?file=curl2.sh"></script>
 
+        <h4>Widgets</h4>
+
+        <p>Dashing comes with few re-usable widgets (reminder: <i>data-view="WidgetName"</i> in dashboard defines the used widget):
```

Comment:
Yes, I'm pretty much fine with any additional change (while still holding the original idea of three items mentioned in PR). So go ahead, you have my blessing :+1: